packages/appstore/src/components/wallet-account/wallet-account.tsx

```diff
@@ -0,0 +1,160 @@
+                            </Text>
+                            <div className='wallet-account__actions'>
+                                <WalletActionButton size='small' label='Transfer' icon='icAppstoreTransfer' />
+                                <WalletActionButton size='small' label='Transaction' icon='icAppstoreTransaction' />
+                                <WalletActionButton size='small' label='Reset Balance' icon='icAppstoreResetBalance' />
+                            </div>
+                        </div>
+                    </div>
+                    <div className='wallet-account__balance'>
+                        {isAccountVirtual && (
+                            <Text bold={false} type='small' align='right'>
+                                {localize('Virtual balance')}
+                            </Text>
+                        )}
+                        <Text bold type='subtitle-1'>
+                            <Money amount={account.wallet?.balance} currency={account.currency} show_currency />
+                        </Text>
+                    </div>
+                </div>
+            </Accordion.Title>
+            <Accordion.Content size='medium'>
+                <div className='wallet-account__content'>
+                    <div>
+                        <Text bold type='subtitle-1'>
+                            {localize('CFDs')}
+                        </Text>
+                        <Text bold={false} type='paragraph-1'>
+                            {localize(
+                                'Trade with leverage and tight spreads for better returns on successful trades. '
+                            )}
+                            <a href='#' className='link-text'>
+                                {localize('Learn more')}
+                            </a>
+                        </Text>
+                    </div>
+
+                    <div className='wallet-account__apps-wrapper'>
+                        <div className='wallet-account__apps-title'>
+                            <Text bold type='paragraph-1'>
+                                {localize('Deriv MT5')}
+                            </Text>
+                            <Button className='wallet-account__apps-title--link' size='small' color='tertiary'>
+                                {localize('Compare accounts')}
+                            </Button>
+                        </div>
+                        <div className='wallet-account__apps '>
+                            <AppsLauncher
+                                app_name='Derived'
+                                icon_name='TradingDerived'
+                                is_app_installed={false}
+                                button_className='wallet-account__app-launcher-button'
+                                jurisdiction=''
+                                description='Trade CFDs on MT5 with synthetic indices that simulate real-world market movements.'
```

Comment:
Do we not need to localize the descriptions?

Reply:
you're right. will add it to other similar props as well :thumbs-up: